src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -496,10 +572,10 @@ export default function SpeedUpAndCancelSheet() {
                   <GasSpeedButtonContainer>
                     <GasSpeedButton
                       currentNetwork={currentNetwork}
-                      minGasPrice={minGasPrice}
+                      minMaxPriorityFeePerGas={minMaxPriorityFeePerGas}
```

Comment:
is `minMaxPriorityFeePerGas` used in `GasSpeedButton`?

Reply:
removed